### ISSUE TITLE
decompression bomb DOS attack Warning

### ISSUE THREAD

**Issue by ravitejarj** [zhanghang1989/PyTorch-Encoding#377]
When i try train the model with 1500 images i am getting this warning and after that training stopped
model = encnet
backbone = resnest101
learning rate = 0.0125 (changed)
base size = 480 (changed)
crop size = 440 (changed)
batch size = 2

/home/hbbg/miniconda3/envs/seg-pe/lib/python3.7/site-packages/PIL/Image.py:2850: DecompressionBombWarning: Image size (133036628 pixels) exceeds limit of 89478485 pixels, could be decompression bomb DOS attack.
  DecompressionBombWarning,
